### ISSUE TITLE
False positive trigger for `enforce-store-naming-convention`

### ISSUE THREAD

**Issue by Kelin2025** [effector/eslint-plugin#136]
Sometimes plugin treats factories from `effector-factorio` as stores

Minimal reproduction code
```ts
import { combine, sample } from 'effector'
import { modelFactory } from 'effector-factorio'

export const factory = modelFactory(() => {
  sample({
    clock: combine([createStore('')]),
    fn: ([fieldType, customFieldId]) => customFieldId || fieldType,
    target: createStore(''),
  })

  return {}
})
```
Result:
<img width="672" alt="Screenshot 2023-03-09 at 20 22 29" src="https://user-images.githubusercontent.com/4208480/224106125-8292cc35-3396-4877-8596-01355326b2ee.png">

Parser: @typescript-eslint/parser (5.53.0)
